Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -2807,4 +2813,55 @@ defineSuite([
         });
     });
 
+    it('clipping planes cull hidden tiles', function() {
+        // Root tile has a content box that is half the extents of its box
+        // Expect to cull root tile and three child tiles
+        return Cesium3DTilesTester.loadTileset(scene, tilesetUrl).then(function(tileset) {
+            var visibility = tileset._root.visibility(scene.frameState, CullingVolume.MASK_INSIDE);
+
+            expect(visibility).not.toBe(CullingVolume.MASK_OUTSIDE);
+
+            var plane = new Plane(Cartesian3.UNIT_Z, 100000000.0);
+            tileset.clippingPlanes = new ClippingPlanesCollection({
+                planes : [
+                    plane
+                ]
+            });
+
+            visibility = tileset._root.visibility(scene.frameState, CullingVolume.MASK_INSIDE);
+
+            expect(visibility).toBe(CullingVolume.MASK_OUTSIDE);
+
+            plane.distance = 0.0;
+            visibility = tileset._root.visibility(scene.frameState, CullingVolume.MASK_INSIDE);
+
+            expect(visibility).not.toBe(CullingVolume.MASK_OUTSIDE);
+        });
```

Comment:
This test should be tweaked to call tileset.update and check the number of draw commands, similar to some of the other tests. It should expect 5 commands before the clipping plane is enabled and 1 command after. Checking the visibility directly is a bit too low level.

Realized later this is basically a duplicate of @pjcozzi's comment.